README.creole

```diff
@@ -101,4 +108,4 @@ package or remove it.
 
 Oh yes. Definitely.
 
-* errrors - when there's an error we don't handle it very gracefully at all.
+* errors - when there's an error we don't handle it very gracefully at all.
```

Comment:
Because I like to think it's your style, i think i've read this somehow in your articles :D 

Cheers,